### ISSUE TITLE
Failed to run Ktor with kotlinx.coroutines 1.5.0-RC-native-mt

### ISSUE THREAD

**Issue by e5l** [Kotlin/kotlinx.coroutines#2686]
Lots of tests fail with InvalidMutabilityException in TimeoutCoroutine:
```
kotlin.IllegalStateException: Test failed with engine: CIO
kotlin.native.concurrent.InvalidMutabilityException: mutation attempt of frozen kotlinx.coroutines.TimeoutCoroutine@d0e2de58
	0   test.kexe                           0x0000000100d3ab9f kfun:kotlin.Throwable#<init>(kotlin.String?){} + 95 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/Throwable.kt:23:37)
	1   test.kexe                           0x0000000100d3329d kfun:kotlin.Exception#<init>(kotlin.String?){} + 93 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/Exceptions.kt:23:44)
	2   test.kexe                           0x0000000100d3347d kfun:kotlin.RuntimeException#<init>(kotlin.String?){} + 93 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/Exceptions.kt:34:44)
	3   test.kexe                           0x0000000100d6b08d kfun:kotlin.native.concurrent.InvalidMutabilityException#<init>(kotlin.String){} + 93 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/concurrent/Freezing.kt:22:60)
	4   test.kexe                           0x0000000100d6c86f ThrowInvalidMutabilityException + 431 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/concurrent/Internal.kt:93:11)
	5   test.kexe                           0x0000000100eeec20 MutationCheck + 128
	6   test.kexe                           0x0000000100f6a7cf kfun:kotlinx.coroutines.TimeoutCoroutine.<init>#internal + 303 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/Timeout.kt:152:5)
	7   test.kexe                           0x0000000100f69684 kfun:kotlinx.coroutines.$withTimeoutOrNullCOROUTINE$8#invokeSuspend(kotlin.Result<kotlin.Any?>){}kotlin.Any? + 1076 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/Timeout.kt:102:36)
	8   test.kexe                           0x0000000100f69b00 kfun:kotlinx.coroutines#withTimeoutOrNull(kotlin.Long;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,0:0>){0§<kotlin.Any?>}0:0? + 352 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/Timeout.kt:96:16)
	9   test.kexe                           0x00000001008de90f kfun:io.ktor.client.engine.cio.Endpoint.$connectCOROUTINE$500.invokeSuspend#internal + 8415 (/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-cio/common/src/io/ktor/client/engine/cio/Endpoint.kt:191:42)
	10  test.kexe                           0x00000001008dee8f kfun:io.ktor.client.engine.cio.Endpoint.connect#internal + 383 (/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-cio/common/src/io/ktor/client/engine/cio/Endpoint.kt:173:21)
	11  test.kexe                           0x00000001008e26ef kfun:io.ktor.client.engine.cio.Endpoint.$makeDedicatedRequest$lambda-3COROUTINE$503.invokeSuspend#internal + 2847 (/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-cio/common/src/io/ktor/client/engine/cio/Endpoint.kt:106:30)
	12  test.kexe                           0x0000000100d5df14 kfun:kotlin.coroutines.native.internal.BaseContinuationImpl#resumeWith(kotlin.Result<kotlin.Any?>){} + 788 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/coroutines/ContinuationImpl.kt:30:39)
	13  test.kexe                           0x0000000100f9f4b7 kfun:kotlinx.coroutines.DispatchedTask#run(){} + 2871 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/internal/DispatchedTask.kt:106:71)
	14  test.kexe                           0x0000000100f47f90 kfun:kotlinx.coroutines.EventLoopImplBase#processNextEvent(){}kotlin.Long + 896 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/EventLoop.common.kt:277:22)
	15  test.kexe                           0x00000001008887d8 kfun:io.ktor.test.dispatcher.$testSuspend$lambda-1COROUTINE$540.invokeSuspend#internal + 1688 (/Users/jetbrains/work/jb/ktor/ktor-test-dispatcher/macosX64/src/TestMacosX64.kt:26:31)
	16  test.kexe                           0x0000000100d5df14 kfun:kotlin.coroutines.native.internal.BaseContinuationImpl#resumeWith(kotlin.Result<kotlin.Any?>){} + 788 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/coroutines/ContinuationImpl.kt:30:39)
	17  test.kexe                           0x0000000100f9f4b7 kfun:kotlinx.coroutines.DispatchedTask#run(){} + 2871 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/internal/DispatchedTask.kt:106:71)
	18  test.kexe                           0x0000000100f47f90 kfun:kotlinx.coroutines.EventLoopImplBase#processNextEvent(){}kotlin.Long + 896 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/EventLoop.common.kt:277:22)
	19  test.kexe                           0x0000000100fc914f kfun:kotlinx.coroutines#runEventLoop(kotlinx.coroutines.EventLoop?;kotlin.Function0<kotlin.Boolean>){} + 831 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/native/src/Builders.kt:80:40)
	20  test.kexe                           0x0000000100fcadee kfun:kotlinx.coroutines.BlockingCoroutine.joinBlocking#internal + 430 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/native/src/Builders.kt:67:9)
	21  test.kexe                           0x0000000100fc8849 kfun:kotlinx.coroutines#runBlocking(kotlin.coroutines.CoroutineContext;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,0:0>){0§<kotlin.Any?>}0:0 + 1273 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/native/src/Builders.kt:50:22)
	22  test.kexe                           0x0000000100fc8d3e kfun:kotlinx.coroutines#runBlocking$default(kotlin.coroutines.CoroutineContext?;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,0:0>;kotlin.Int){0§<kotlin.Any?>}0:0 + 350 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/native/src/Builders.kt:33:15)
	23  test.kexe                           0x00000001008873ea kfun:io.ktor.test.dispatcher#testSuspend(kotlin.coroutines.CoroutineContext;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,kotlin.Unit>){} + 250 (/Users/jetbrains/work/jb/ktor/ktor-test-dispatcher/macosX64/src/TestMacosX64.kt:20:11)
	24  test.kexe                           0x0000000100887571 kfun:io.ktor.test.dispatcher#testSuspend$default(kotlin.coroutines.CoroutineContext?;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,kotlin.Unit>;kotlin.Int){} + 305 (/Users/jetbrains/work/jb/ktor/ktor-test-dispatcher/macosX64/src/TestMacosX64.kt:17:15)
	25  test.kexe                           0x000000010089363f kfun:io.ktor.client.tests.utils#testWithEngine(io.ktor.client.engine.HttpClientEngineFactory<0:0>;io.ktor.client.tests.utils.ClientLoader?;kotlin.coroutines.SuspendFunction1<io.ktor.client.tests.utils.TestClientBuilder<0:0>,kotlin.Unit>){0§<io.ktor.client.engine.HttpClientEngineConfig>} + 335 (/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-tests/common/src/io/ktor/client/tests/utils/CommonClientTestUtils.kt:66:5)
	26  test.kexe                           0x00000001008937bf kfun:io.ktor.client.tests.utils#testWithEngine$default(io.ktor.client.engine.HttpClientEngineFactory<0:0>;io.ktor.client.tests.utils.ClientLoader?;kotlin.coroutines.SuspendFunction1<io.ktor.client.tests.utils.TestClientBuilder<0:0>,kotlin.Unit>;kotlin.Int){0§<io.ktor.client.engine.HttpClientEngineConfig>} + 303 (/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-tests/common/src/io/ktor/client/tests/utils/CommonClientTestUtils.kt:62:8)
	27  test.kexe                           0x000000010089b9ee kfun:io.ktor.client.tests.utils.ClientLoader#clientTests(kotlin.collections.List<kotlin.String>;kotlin.coroutines.SuspendFunction1<io.ktor.client.tests.utils.TestClientBuilder<io.ktor.client.engine.HttpClientEngineConfig>,kotlin.Unit>){} + 3566 (/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-tests/posix/src/io/ktor/client/tests/utils/ClientLoaderNative.kt:42:17)
	28  test.kexe                           0x000000010089bfda kfun:io.ktor.client.tests.utils.ClientLoader#clientTests$default(kotlin.collections.List<kotlin.String>?;kotlin.coroutines.SuspendFunction1<io.ktor.client.tests.utils.TestClientBuilder<io.ktor.client.engine.HttpClientEngineConfig>,kotlin.Unit>;kotlin.Int){} + 362 (/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-tests/posix/src/io/ktor/client/tests/utils/ClientLoaderNative.kt:27:19)
	29  test.kexe                           0x000000010090af17 kfun:io.ktor.client.tests.AttributesTest#testKeepAttributes(){} + 199 (/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/AttributesTest.kt:15:32)
	at kotlin.Throwable#<init>(/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/Throwable.kt:23)
	at kotlin.Exception#<init>(/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/Exceptions.kt:23)
	at kotlin.RuntimeException#<init>(/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/Exceptions.kt:34)
	at kotlin.IllegalStateException#<init>(/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/Exceptions.kt:70)
	at io.ktor.client.tests.utils.ClientLoader#clientTests(/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-tests/posix/src/io/ktor/client/tests/utils/ClientLoaderNative.kt:58)
	at io.ktor.client.tests.utils.ClientLoader#clientTests$default(/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-tests/posix/src/io/ktor/client/tests/utils/ClientLoaderNative.kt:27)
	at io.ktor.client.tests.AttributesTest#testKeepAttributes(/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/AttributesTest.kt:15)
	at io.ktor.client.tests.$AttributesTest$test$0.$testKeepAttributes$FUNCTION_REFERENCE$3.invoke#internal(/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/AttributesTest.kt:15)
	at io.ktor.client.tests.$AttributesTest$test$0.$testKeepAttributes$FUNCTION_REFERENCE$3.$<bridge-UNNN>invoke(/Users/jetbrains/work/jb/ktor/ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/AttributesTest.kt:15)
	at kotlin.native.internal.test.BaseClassSuite.TestCase#run(/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/internal/test/TestSuite.kt:85)
	at kotlin.native.internal.test.TestRunner.run#internal(/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/internal/test/TestRunner.kt:245)
	at kotlin.native.internal.test.TestRunner.runIteration#internal(/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/internal/test/TestRunner.kt:271)
	at kotlin.native.internal.test.TestRunner#run(/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/internal/test/TestRunner.kt:286)
	at kotlin.native.internal.test#testLauncherEntryPoint(/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/internal/test/Launcher.kt:22)
	at kotlin.native.internal.test#main(/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/internal/test/Launcher.kt:26)
	at <global>.Konan_start(/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/internal/test/Launcher.kt:25)
	at <global>.Init_and_run_start(Unknown Source)
	at <global>.start(Unknown Source)
	at <global>.0x0(Unknown Source)
```

**Comment by whyoleg** [comment]
Same here with rsocket-kotlin (macos target):
```stacktrace
kotlin.native.concurrent.InvalidMutabilityException: mutation attempt of frozen kotlinx.coroutines.TimeoutCoroutine@45d407b8
    at 0   test.kexe                           0x0000000100a3158f kfun:kotlin.Throwable#<init>(kotlin.String?){} + 95 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/Throwable.kt:23:37)
    at 1   test.kexe                           0x0000000100a2a76d kfun:kotlin.Exception#<init>(kotlin.String?){} + 93 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/Exceptions.kt:23:44)
    at 2   test.kexe                           0x0000000100a2a94d kfun:kotlin.RuntimeException#<init>(kotlin.String?){} + 93 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/Exceptions.kt:34:44)
    at 3   test.kexe                           0x0000000100a5fa4d kfun:kotlin.native.concurrent.InvalidMutabilityException#<init>(kotlin.String){} + 93 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/concurrent/Freezing.kt:22:60)
    at 4   test.kexe                           0x0000000100a6115f ThrowInvalidMutabilityException + 431 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/concurrent/Internal.kt:93:11)
    at 5   test.kexe                           0x0000000100ddaaa0 MutationCheck + 128
    at 6   test.kexe                           0x0000000100b639bf kfun:kotlinx.coroutines.TimeoutCoroutine.<init>#internal + 303 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/Timeout.kt:152:5)
    at 7   test.kexe                           0x0000000100b633d5 kfun:kotlinx.coroutines#withTimeout(kotlin.Long;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,0:0>){0§<kotlin.Any?>}0:0 + 341 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/Timeout.kt:44:22)
    at 8   test.kexe                           0x0000000100b631fa kfun:kotlinx.coroutines#withTimeout(kotlin.time.Duration;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,0:0>){0§<kotlin.Any?>}0:0 + 202 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/Timeout.kt:72:12)
    at 9   test.kexe                           0x0000000100cedbc2 kfun:io.rsocket.kotlin.test.SuspendTest.$runPhaseCOROUTINE$2.invokeSuspend#internal + 1570 (/Users/whyoleg/projects/rsocket/rsocket-kotlin/rsocket-test/src/commonMain/kotlin/io/rsocket/kotlin/test/SuspendTest.kt:71:13)
    at 10  test.kexe                           0x0000000100cee54b kfun:io.rsocket.kotlin.test.SuspendTest.runPhase#internal + 507 (/Users/whyoleg/projects/rsocket/rsocket-kotlin/rsocket-test/src/commonMain/kotlin/io/rsocket/kotlin/test/SuspendTest.kt:68:21)
    at 11  test.kexe                           0x0000000100cf0904 kfun:io.rsocket.kotlin.test.SuspendTest.$test$lambda-2COROUTINE$5.invokeSuspend#internal + 2916 (/Users/whyoleg/projects/rsocket/rsocket-kotlin/rsocket-test/src/commonMain/kotlin/io/rsocket/kotlin/test/SuspendTest.kt:47:26)
    at 12  test.kexe                           0x0000000100a539b4 kfun:kotlin.coroutines.native.internal.BaseContinuationImpl#resumeWith(kotlin.Result<kotlin.Any?>){} + 788 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/coroutines/ContinuationImpl.kt:30:39)
    at 13  test.kexe                           0x0000000100beaca3 kfun:kotlinx.coroutines.internal.ShareableContinuation#resumeWith(kotlin.Result<1:0>){} + 595 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/native/src/internal/Sharing.kt:185:22)
    at 14  test.kexe                           0x0000000100bc214e kfun:kotlinx.coroutines.internal.ScopeCoroutine#afterResume(kotlin.Any?){} + 366 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/internal/Scopes.kt:42:15)
    at 15  test.kexe                           0x0000000100b25dd4 kfun:kotlinx.coroutines.AbstractCoroutine#resumeWith(kotlin.Result<1:0>){} + 452 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/AbstractCoroutine.kt:106:9)
    at 16  test.kexe                           0x0000000100a53ca5 kfun:kotlin.coroutines.native.internal.BaseContinuationImpl#resumeWith(kotlin.Result<kotlin.Any?>){} + 1541 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/coroutines/ContinuationImpl.kt:43:32)
    at 17  test.kexe                           0x0000000100bb92ba kfun:kotlinx.coroutines.DispatchedTask#run(){} + 3002 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/internal/DispatchedTask.kt:106:71)
    at 18  test.kexe                           0x0000000100b41b60 kfun:kotlinx.coroutines.EventLoopImplBase#processNextEvent(){}kotlin.Long + 896 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/common/src/EventLoop.common.kt:277:22)
    at 19  test.kexe                           0x0000000100bdd692 kfun:kotlinx.coroutines#runEventLoop(kotlinx.coroutines.EventLoop?;kotlin.Function0<kotlin.Boolean>){} + 898 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/native/src/Builders.kt:80:40)
    at 20  test.kexe                           0x0000000100bdc94e kfun:kotlinx.coroutines.BlockingCoroutine.joinBlocking#internal + 430 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/native/src/Builders.kt:67:9)
    at 21  test.kexe                           0x0000000100bdba5d kfun:kotlinx.coroutines#runBlocking(kotlin.coroutines.CoroutineContext;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,0:0>){0§<kotlin.Any?>}0:0 + 1293 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/native/src/Builders.kt:50:22)
    at 22  test.kexe                           0x0000000100bdbf8b kfun:kotlinx.coroutines#runBlocking$default(kotlin.coroutines.CoroutineContext?;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,0:0>;kotlin.Int){0§<kotlin.Any?>}0:0 + 347 (/opt/buildAgent/work/44ec6e850d5c63f0/kotlinx-coroutines-core/native/src/Builders.kt:33:15)
    at 23  test.kexe                           0x0000000100d21ed0 kfun:io.rsocket.kotlin.test#runTest(kotlin.Boolean;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,kotlin.Unit>){} + 176 (/Users/whyoleg/projects/rsocket/rsocket-kotlin/rsocket-test/src/nativeMain/kotlin/io/rsocket/kotlin/test/Test.kt:28:5)
    at 24  test.kexe                           0x0000000100cecbc4 kfun:io.rsocket.kotlin.test.SuspendTest#test(kotlin.time.Duration;kotlin.Boolean;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,kotlin.Unit>){} + 340 (/Users/whyoleg/projects/rsocket/rsocket-kotlin/rsocket-test/src/commonMain/kotlin/io/rsocket/kotlin/test/SuspendTest.kt:38:9)
    at 25  test.kexe                           0x0000000100cece83 kfun:io.rsocket.kotlin.test.SuspendTest#test$default(kotlin.time.Duration?;kotlin.Boolean;kotlin.coroutines.SuspendFunction1<kotlinx.coroutines.CoroutineScope,kotlin.Unit>;kotlin.Int){} + 611 (/Users/whyoleg/projects/rsocket/rsocket-kotlin/rsocket-test/src/commonMain/kotlin/io/rsocket/kotlin/test/SuspendTest.kt:34:5)
    at 26  test.kexe                           0x0000000100cf5acd kfun:io.rsocket.kotlin.test.TransportTest#requestChannel0(){} + 285 (/Users/whyoleg/projects/rsocket/rsocket-kotlin/rsocket-test/src/commonMain/kotlin/io/rsocket/kotlin/test/TransportTest.kt:59:29)
    at 27  test.kexe                           0x0000000100d24e88 kfun:io.rsocket.kotlin.transport.local.$LocalTransportTest$test$0.$requestChannel0$FUNCTION_REFERENCE$4.invoke#internal + 72 (/Users/whyoleg/projects/rsocket/rsocket-kotlin/rsocket-transport-local/src/commonTest/kotlin/io/rsocket/kotlin/transport/local/LocalTransportTest.kt:38:628)
    at 28  test.kexe                           0x0000000100d250c1 kfun:io.rsocket.kotlin.transport.local.$LocalTransportTest$test$0.$requestChannel0$FUNCTION_REFERENCE$4.$<bridge-UNNN>invoke(-1:0){}#internal + 97 (/Users/whyoleg/projects/rsocket/rsocket-kotlin/rsocket-transport-local/src/commonTest/kotlin/io/rsocket/kotlin/transport/local/LocalTransportTest.kt:38:628)
    at 29  test.kexe                           0x0000000100a8c224 kfun:kotlin.native.internal.test.BaseClassSuite.TestCase#run(){} + 1316 (/Users/teamcity1/teamcity_work/290aee0e088a1666/runtime/src/main/kotlin/kotlin/native/internal/test/TestSuite.kt:85:26)

```

**Comment by ArcticLampyrid** [comment]
Seems caused by #2688

**Comment by qwwdfsad** [comment]
Fixed in 1.5.0-native-mt
